src/renderers/testing/ReactTestRenderer.js

```diff
@@ -53,20 +53,23 @@ class ReactTestComponent {
   _currentElement: ReactElement;
   _renderedChildren: null | Object;
   _topLevelWrapper: null | ReactInstance;
+  _hostContainerInfo: null | Object;
 
   constructor(element: ReactElement) {
     this._currentElement = element;
     this._renderedChildren = null;
     this._topLevelWrapper = null;
+    this._hostContainerInfo = null;
   }
 
   mountComponent(
     transaction: ReactTestReconcileTransaction,
     nativeParent: null | ReactTestComponent,
-    nativeContainerInfo: ?null,
+    hostContainerInfo: null | Object,
```

Comment:
Doesn't it always exist in test renderer? We probably shouldn't check for its existence below as well.